src/kinova_gen3_mujoco_config/config/config.yaml

```diff
@@ -0,0 +1,228 @@
+###############################################################
+#
+#  This configures the robot to work with MoveIt Pro
+#
+###############################################################
+
+# Baseline hardware configuration parameters for MoveIt Pro.
+# [Required]
+hardware:
+  # Used by the description package to set kinematics and geometry for a specific robot type.
+  # [Required]
+  type: "kinova_gen3_mujoco"
+
+  # Set simulated to false if you are using this as a configuration for real hardware.
+  # Override-able by setting the $MOCK_HARDWARE environment variable to 'True'.
+  # This allows users to switch between mock and real hardware using the same configuration.
+  # [Required]
+  simulated: ${MOCK_HARDWARE:-true}
+
+  # If the MoveIt Pro Agent should launch the ros2 controller node.
+  # [Optional, default=True]
+  launch_control_node: True
+
+  # If the MoveIt Pro Agent should launch the robot state publisher.
+  # This should be false if you are launching the robot state publisher as part of drivers.
+  # [Optional, default=True]
+  launch_robot_state_publisher: True
+
+  # The robot's IP address.
+  # [Required]
+  ip: "0.0.0.0"
+
+  # Specify additional launch files for running the robot with real hardware.
+  # [Optional, defaults to a blank launch file if not specified]
+  robot_driver_persist_launch_file:
+    package: "kinova_gen3_mujoco_config"
+    path: "launch/robot_drivers_to_persist.launch.py"
+  hardware_launch_file:
+    package: "moveit_studio_agent"
+    path: "launch/blank.launch.py"
+
+  # Specify any additional launch files for running the robot in simulation mode.
+  # Used when hardware.simulated is True.
+  # [Optional, defaults to a blank launch file if not specified]
+#  simulated_robot_driver_persist_launch_file:
+#    package: "kinova_gen3_mujoco_config"
+#    path: "launch/sim/robot_drivers_to_persist_sim.launch.py"
+#  simulated_hardware_launch_file:
+#    package: "moveit_studio_agent"
+#    path: "launch/blank.launch.py"
+
+  # Parameters used to configure the robot description through XACRO.
+  # A URDF and SRDF are both required.
+  # [Required]
+  robot_description:
+    urdf:
+      package: "kinova_gen3_mujoco_config"
+      path: "description/picknik_kinova_gen3.xacro"
+    srdf:
+      package: "kinova_gen3_mujoco_config"
+      path: "config/moveit/picknik_kinova_gen3_base.srdf"
+    # Specify any additional parameters required for the URDF.
+    # Many of these are specific to the UR descriptions packages, and can be customized as needed.
+    # [Optional]
+    urdf_params:
+      - arm: "gen3"
+      - dof: "7"
+      - vision: "false"
+      - robot_ip: "%>> hardware.ip"
+      - gripper: "robotiq_2f_85"
+      - username: "admin"
+      - password: "admin"
```

Comment:
What are these needed for?

Reply:
These were  needed for the kinova hardware interface configuration, removing them and some others since they aren't relevant for this config